src/Internal/TypeTokenFactory.php

```diff
@@ -100,6 +100,11 @@ public function create(
             }
         }
 
+        if ($property !== null && $property->getType() !== null) {
```

Comment:
You can check to see if `$property` has the method `getType()` and support back to 7.1